lib/ipmi_vita.c

```diff
@@ -950,7 +903,7 @@ ipmi_vita_main (struct ipmi_intf *intf, int argc, char **argv)
 	case VITA_CMD_POLICY_GET:
 		if (argc > 2) {
 			rc = ipmi_vita_get_fru_state_policy_bits(intf,
-				&argv[2]);
+								 &argv[2]);
```

Comment:
Could as well invert the condition and reduce indentation:
```
if (argc <= 2) {
    break;
}

rc = ipmi_vita...
break;
```

Reply:
In this case, i just bumped the argv[2] onto the same line as the call.  It goes over 80 but it's close enough.